mobile-widgets/themeinterface.cpp

```diff
@@ -16,6 +19,18 @@ void themeInterface::setup(QQmlContext *ct)
 	// get current theme
 	instance()->m_currentTheme = qPrefDisplay::theme();
 	instance()->update_theme();
+
+	// check system font
+	instance()->m_basePointSize = defaultModelFont().pointSize();
+	qDebug() << "Using the following font:" <<
+				defaultModelFont().family() <<
+				"at" <<
+				instance()->m_basePointSize << "pt" <<
+				"with mobile_scale:" <<
+				qPrefDisplay::mobile_scale();
```

Comment:
see above... let's use `appendTextToLog()`. That gives us nice time stamps and some consistency in the log.

Reply:
I will use appendTextLog in the future here it is removed.